Gemfile

```diff
@@ -2,6 +2,5 @@ source "https://rubygems.org"
 
 gemspec
 
-gem 'rails', github: 'rails/rails'
-gem 'arel',  github: 'rails/arel'
+gem 'activemodel', '>= 5.x'
 gem 'mocha'
```

Comment:
Hm, with mocha I don't care so much, but since Active Model shares mostly the same version requirement as railties, I'd shove it in the spec. Of course, mocha is the only dev dependency left, so I'd move that too. But that's what I'd do :grin: